ckanext/canada/templates/package/base.html

```diff
@@ -7,7 +7,6 @@
   {% set pkg_url = h.url_for(pkg.type ~ '.read', id=pkg.id if is_activity_archive else pkg.name, **({'activity_id': request.args['activity_id']} if 'activity_id' in request.args else {})) %}
   {% if g.action != 'new' and pkg %}
     {% set dataset = h.get_translated(pkg, 'title') %}
-    <li><a href="{{ pkg_url }}">{{ dataset|truncate(30) }}</a></li>
+    <li><a href="{{ pkg_url }}" title="{{ dataset }}">{{ dataset|truncate(30) }}</a></li>
```

Comment:
@RabiaSajjad increase the `truncate` part here to more than `30`? or nah?

Reply:
I can change it 80 so that it's the same everywhere.